README.md

```diff
@@ -17,7 +17,7 @@ like Arduino on Facebook at: https://www.facebook.com/official.arduino
 http://github.com/arduino/Arduino/issues
 
 * More complex requests and technical discussion should go on the Arduino Developers
-mailing list:
+Mailing list:
```

Comment:
'mailing list' is correct in this case (read the whole sentence: goes from line 19 to 20)